framework/src/kernels/MatReaction.C

```diff
@@ -19,7 +19,10 @@ MatReaction::validParams()
                        "Set this to make v a coupled variable, otherwise it will use the "
                        "kernel's nonlinear variable for v");
   params.addClassDescription("Kernel to add -L*v, where L=reaction rate, v=variable");
-  params.addParam<MaterialPropertyName>("mob_name", "L", "The reaction rate used with the kernel");
+  params.addDeprecatedParam<MaterialPropertyName>(
```

Comment:
```suggestion
  params.addRequiredParam<MaterialPropertyName>(
```

Reply:
It's not required (but should be) instead it has a default of `L`